geometry_msgs/PoseStamped pose
string target_frame
---
geometry_msgs/PoseStamped transformed_pose